gettsim/parameters/arbeitsl_geld_2.yaml

```diff
@@ -380,3 +392,40 @@ abzugsfähige_pausch:
     werbung: 15.33
     reference: V. v. 21.06.2011 BGBl. I S. 1175.
     note: Fester Betrag. Betrag unverändert. Warum haben wir das? Es ändert sich nichts
+
+max_miete_pro_qm:
```

Comment:
Note that these parameters do not reflect actual law, but general rules how 'Arbeitsagenturen' apparently work. They don't make sense before 2005-01-01 anyway. I have a feeling the 10€ rule in particular might be different by now in light of inflation. The whole complex probably deserves an own issue.

Reply:
Sounds reasonable. Let's make a note in the description of this parameter that it is a commonly used rule of thumb by Arbeitsagenturen instead of proper law and address this in a later PR properly.